### PR TITLE
Css canvas medical

### PR DESCRIPTION
**Pull request by Jc-Leon**
Screen Shot 2021-06-30 at 8 48 23 AM
Screen Shot 2021-06-30 at 8 48 43 AM
Screen Shot 2021-06-30 at 8 48 54 AM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey dude!! The position of your text over the hero image is not matching the position on the example, please revise

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Jc! I see you are using text for the logo rather than the images provided, please switch those over to being the logo images provided in the images folder. Make sure you attach updated screenshots after making the change! 😁

**Comment by Jc-Leon** [comment]
Screen Shot 2021-07-08 at 10 44 45 PM
Screen Shot 2021-07-08 at 10 44 41 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey dude! Can I see updated screenshots for your entire output after the changes I've requested? I want to see the image portion too, and re-check if the rest matches the example.

**Comment by Jc-Leon** [comment]
Screen Shot 2021-07-09 at 1 19 59 PM
Screen Shot 2021-07-09 at 1 20 05 PM
Screen Shot 2021-07-09 at 1 19 22 PM

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  background-color: #525351;
+  color: white;
+  padding: 7rem;
+  flex-basis: 25%;
+  letter-spacing: 1px;
+}
+.address h2 {
+    padding-bottom: 15px;
+    font-family: 'Tiempos Headline';
+    font-size: 35px;p
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:147]: typo

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  flex-direction: column;
+  position: absolute;
+  width: 36rem;
+  padding-left: 3rem;
+  bottom: 56%;
+  color: white;
+}
+.hero-text button {
+  position: absolute;
+  bottom: -21rem;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:76]: You _definitely_ don't want to use negative values for things like this. What is the purpose of this? Did you make the parent element of this `relative` so you can confine this element to its containing block?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey dude! I gave you a couple notes, but I have a couple more broad notes for you:
1. All of your sizing in this is on an element-by-element individual basis, there is no use of layout classes at all, please revise this to make use of layout classes so your styling can be more succinct
2. There is a lot of repetition of styling throughout your stylesheet, try to find instances of styling that is heavily repeated and create a class for that specific styling that you can apply to any elements that need that styling. That way, you can get the same effect with a much more trim stylesheet

**Pull request merged**